config/bootstrap.php

```diff
@@ -21,6 +21,7 @@
 if (PHP_VERSION_ID < 70000) {
     class_alias('Cake\Utility\Text', 'Cake\Utility\String');
 }
+class_alias('Cake\Event\EventDispatcherTrait', 'Cake\Event\EventManagerTrait');
```

Comment:
This should be in bootstrap.php or functions.php in `src/Event` as Event package is available standalone.